src/style.css

```diff
@@ -0,0 +1,189 @@
+/* #region Reset */
+:root {
+  box-sizing: border-box;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  --Main-accent: #060b35;
+  --Secondary: #616070;
+  --Blue-accent: #00acdc;
+
+}
+
+body {
+  margin: 0;
+}
+/* #endregion */
+/* #region Card */
+.card {
+  position: relative;
+  left: 0;
+  top: 0;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-photo {
+  position: absolute;
+  left: 19px;
+  top: 32px;
+  width: 160px;
+  height: 134px;
+  background-image: url("images/imac.jpeg");
+  background-position: center;
+  background-size: 160px 134px;
+}
+
+.card__product-title {
+  text-decoration: none;
+  position: absolute;
+  width: 166px;
+  left: 16px;
+  top: 206px;
+  color: #060b35;
+}
```

Comment:
it's a good practice to avoid position absolute, here you can just use margins to set suitable spaces for all elements
https://dev.to/domagojvidovic/position-absolute-and-relative-the-simplest-guide-5169

When To Use It
If you need to adjust the layout within a page or a component, you should avoid position: absolute; in 99% of the cases. Opt-in for Flexbox or Grid instead.

However, if you want to add a badge to your image, it's just perfect. You always want the badge to be on the top (or the edge) of the image, so that it seems like it's a part of an image.

Or if you want to stack a few images/elements onto each other. There's no way other than position: absolute;.

And that's the ultimate guide: use absolute positioning only when there's no other way to achieve that.